synapse/tools/autodoc.py

```diff
@@ -24,9 +26,17 @@
 
 logger = logging.getLogger(__name__)
 
+# src / name / target
+EdgeDef = Tuple[Union[str, None], str, Union[str, None]]
+EdgeDict = Dict[str, Union[str, ]]
```

Comment:
Was this value meant to be a Union with something else besides str?

Reply:
Right now we only use a `doc` key in our edge definitions. Left it as `str, str`